spec/controllers/state_file/questions/ny_county_controller_spec.rb

```diff
@@ -31,6 +31,30 @@
           }
         }
       end
+
+      context "with return_to_review param set" do
+        it "navigates to the state review screen" do
+          post :update, params: form_params.merge({return_to_review: "y"})
+          expect(response).to redirect_to(controller: "ny_school_district", action: :edit, us_state: "ny", return_to_review: 'y')
+        end
+      end
+
+      context "without return_to_review_param set" do
+        it "navigates to the next page in the flow" do
+          post :update, params: form_params
+          controllers = Navigation::StateFileNyQuestionNavigation::FLOW.to_a
```

Comment:
Do we really need a .to_a here? Seems like  `Navigation::StateFileNyQuestionNavigation::FLOW` already returns an array.

Reply:
Good catch - removed